### ISSUE TITLE
[Reactorcraft] Waste disposal drum does not accept more than one stack per kind

### ISSUE THREAD

**Issue by Kobra331** [ReikaKalseki/Reika_Mods_Issues#1720]
Hi,

the waste Disposal Drums only accept one stack(16) Waste per type, so if you have only 64 Technetium waste you need 4 disposal drums to store it, even if the other slots are empty.
If there are more than 16 waste per type in one drum stored, they spam array out of bound exceptions.

**Comment by JujuLynx** [comment]
I can confirm I'm having this issue as well. I upgraded my version from 18b to 19c (along with DragonAPI) and came across it. In addition, after the first 16-item stack is inserted, any following stacks won't flow down into drums placed below it. The first stack will however.

So I reverted back to 18b for RotaryCraft, ReactorCraft and DragonAPI, and I observed some more interesting behaviour. I left the drums that I placed in 19c, and items started flowing down as they should be. However they were only populating 4 items into each slot, instead of 16. Not sure if that's helpful in any way, but as far as the code goes it might be relevant. Breaking and re-placing the drums in 18b returns their original behaviour, with proper flow and stacking.

On a related note, I've observed that Ruthenium-106 is a valid input into the drum despite having a half-life of 373.6 days.
